cmds/dhclient/dhcp6client/packetSock.go

```diff
@@ -0,0 +1,146 @@
+package dhcp6client
+
+import (
+	"encoding/binary"
+	"math/rand"
+	"net"
+
+	"github.com/mdlayher/dhcp6"
+	"golang.org/x/net/ipv6"
+	"golang.org/x/sys/unix"
+)
+
+const (
+	ipv6HdrLen = 40
+	udpHdrLen  = 8
+
+	srcPort = 546
+	dstPort = 547
+)
+
+type packetSock struct {
+	fd      int
+	ifindex int
+}
+
+var bcastMAC = []byte{0x33, 0x33, 0x00, 0x01, 0x00, 0x02}
```

Comment:
What is this bcast mac? What is it for? it's not the mac address of the machine right?